### ISSUE TITLE
image inside pdf

### ISSUE THREAD

**Issue by deepdil-sp** [dbashford/textract#183]
I have a pdf which is an image inside the pdf. It doesn't log me an error. just write as null

**Comment by dbashford** [comment]
Can I get an example PDF?

**Comment by Arapahoe** [comment]
I have the same issue. I have attached a sample image pdf.
[sample.pdf](https://github.com/dbashford/textract/files/3069679/sample.pdf)

**Comment by AlexTurdean** [comment]
Yes. Support for this would be soooo nice. Anyway, amazing job @dbashford! Looking forward for updates :D

**Comment by mrpeppapig** [comment]
Did you try this? 

> textract image.png --tesseract.lang=deu

**Comment by ghost** [comment]
Can you make default pdf method tesseract?
